pandas/core/strings.py

```diff
@@ -311,22 +311,25 @@ def str_replace(arr, pat, repl, n=-1, case=True, flags=0):
 
     Parameters
     ----------
-    pat : string
-        Character sequence or regular expression
+    pat : string or compiled regex
+        String can be a character sequence or regular expression. Also, a
+        compiled regex may be used.
     repl : string or callable
         Replacement string or a callable. The callable is passed the regex
         match object and must return a replacement string to be used.
         See :func:`re.sub`.
 
         .. versionadded:: 0.20.0
+            `pat` also accepts a compiled regex.
```

Comment:
this is not in the right place. you can either create another versionadded for ``pat``